source/manual/data-gov-uk-2nd-line.html.md

```diff
@@ -418,7 +418,9 @@ ckan dataset purge DATASET_NAME
 
 ### Bulk deleting datasets
 
-This needs to be done on the CKAN machine, since [the deletion API is protected from external access][ckan-api-404]. Your API key is required, which can be obtained from your user profile in the CKAN web interface. Put a list of dataset slugs or GUIDs in a text file, with one dataset per line, then run the following.
+This needs to be done on the CKAN machine, since [the deletion API is protected from external access][ckan-api-404]. Your API key is required, which can be obtained from your user profile in the CKAN web interface. You might need to change [the limit to the number of datasets that can be deleted](https://github.com/alphagov/datagovuk_publish/blob/f46b6adf355050a61054b2200f011a6a65a923f4/app/lib/ckan/v26/depaginator.rb#L5).
```

Comment:
Link should probably be https://github.com/alphagov/datagovuk_publish/blob/main/app/lib/ckan/v26/depaginator.rb#L5